### PR TITLE
fix config warnings

### PR DESCRIPTION
**Pull request by hjmodi**

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -1,34 +1,7 @@
 # Quarkus will choose application.yaml over application.properties.
 quarkus:
-  http:
-    port: 8080
-    read-timeout: 30m
-    limits:
-      max-body-size: 500M
```
Comment (ligangty) [inline src/main/resources/application.yaml:7]: Is this causing any warnings?

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -1,34 +1,7 @@
...
   package:
     type: uber-jar
-  application:
-    name: "indy-tracking-service"
-  # Logging (disable console on prod)
```
Comment (ligangty) [inline src/main/resources/application.yaml:12]: Same as above.

**Review by ligangty — COMMENTED** [review]
I'm little confused which type of warnings these configurations are bringing?

**Comment by hjmodi** [comment]
@ligangty please consider the coverage requirement waived because it is caused by fewer lines and not because of less coverage.

File: src/main/resources/application.yaml (hunk 2 above)
Comment (ligangty) [reply src/main/resources/application.yaml:12]: I'd recommend to keep this as a general application name.

File: src/main/resources/application.yaml (hunk 1 above)
Comment (ligangty) [reply src/main/resources/application.yaml:7]: Ok.

**Comment by ligangty** [comment]
> > I'm little confused which type of warnings these configurations are bringing?
> 
> ```
>  - quarkus.resteasy.gzip.enabled is set to 'true' but it is build time fixed to 'false'. Did you change the property quarkus.resteasy.gzip.enabled after building the application?
>  - quarkus.log.min-level is set to 'TRACE' but it is build time fixed to 'DEBUG'. Did you change the property quarkus.log.min-level after building the application?
>  - quarkus.resteasy.gzip.max-input is set to '64M' but it is build time fixed to '10M'. Did you change the property quarkus.resteasy.gzip.max-input after building the application?```
> ```

I think these 3 logs are not fixed by your configurations changes, as they are using default value in build time.

File: src/main/resources/application.yaml (hunk 2 above)
Comment (hjmodi) [reply src/main/resources/application.yaml:12]: added it back

**Comment by hjmodi** [comment]
> > > I'm little confused which type of warnings these configurations are bringing?
> > 
> > 
> > ```
> >  - quarkus.resteasy.gzip.enabled is set to 'true' but it is build time fixed to 'false'. Did you change the property quarkus.resteasy.gzip.enabled after building the application?
> >  - quarkus.log.min-level is set to 'TRACE' but it is build time fixed to 'DEBUG'. Did you change the property quarkus.log.min-level after building the application?
> >  - quarkus.resteasy.gzip.max-input is set to '64M' but it is build time fixed to '10M'. Did you change the property quarkus.resteasy.gzip.max-input after building the application?```
> > ```
> 
> I think these 3 logs are not fixed by your configurations changes, as they are using default value in build time.

I think this is where the run time values are coming from and this change will fix it. Also, it'll be nice to keep only the values we need and remove everything extra from here.

**Review by ligangty — APPROVED** [review]
LGTM

**Pull request merged**